plugins/inputs/procstat/README.md

```diff
@@ -64,19 +71,49 @@ I/O related measurement names (*telegraf* needs to run as **root**):
 - procstat_[prefix_]write_bytes value=1
 
 CPU related measurement names:
-- procstat_[prefix_]cpu_user value=0
-- procstat_[prefix_]cpu_system value=0.01
-- procstat_[prefix_]cpu_idle value=0
-- procstat_[prefix_]cpu_nice value=0
-- procstat_[prefix_]cpu_iowait value=0
-- procstat_[prefix_]cpu_irq value=0
-- procstat_[prefix_]cpu_soft_irq value=0
-- procstat_[prefix_]cpu_soft_steal value=0
-- procstat_[prefix_]cpu_soft_stolen value=0
-- procstat_[prefix_]cpu_soft_guest value=0
-- procstat_[prefix_]cpu_soft_guest_nice value=0
+- procstat_[prefix_]cpu_time value=0.01
+- procstat_[prefix_]cpu_time_user value=0
+- procstat_[prefix_]cpu_time_system value=0.01
+- procstat_[prefix_]cpu_time_idle value=0
+- procstat_[prefix_]cpu_time_nice value=0
+- procstat_[prefix_]cpu_time_iowait value=0
+- procstat_[prefix_]cpu_time_irq value=0
+- procstat_[prefix_]cpu_time_soft_irq value=0
+- procstat_[prefix_]cpu_time_steal value=0
+- procstat_[prefix_]cpu_time_stolen value=0
+- procstat_[prefix_]cpu_time_guest value=0
+- procstat_[prefix_]cpu_time_guest_nice value=0
 
 Memory related measurement names:
 - procstat_[prefix_]memory_rss value=1777664
 - procstat_[prefix_]memory_vms value=24227840
 - procstat_[prefix_]memory_swap value=282624
+- procstat_[prefix_]memory_data value=229376
+- procstat_[prefix_]memory_stack value=135168
+- procstat_[prefix_]memory_locked value=0
+
+Resource limits:
+- procstat_[prefix_]rlimit_cpu_time_hard value=2147483647
+- procstat_[prefix_]rlimit_cpu_time_soft value=2147483647
+- procstat_[prefix_]rlimit_file_locks_hard value=2147483647
+- procstat_[prefix_]rlimit_file_locks_soft value=2147483647
+- procstat_[prefix_]rlimit_memory_data_hard value=2147483647
+- procstat_[prefix_]rlimit_memory_data_soft value=2147483647
+- procstat_[prefix_]rlimit_memory_locked_hard value=65536
+- procstat_[prefix_]rlimit_memory_locked_soft value=65536
+- procstat_[prefix_]rlimit_memory_rss_hard value=2147483647
+- procstat_[prefix_]rlimit_memory_rss_soft value=2147483647
+- procstat_[prefix_]rlimit_memory_stack_hard value=2147483647
+- procstat_[prefix_]rlimit_memory_stack_soft value=8388608
+- procstat_[prefix_]rlimit_memory_vms_hard value=2147483647
+- procstat_[prefix_]rlimit_memory_vms_soft value=2147483647
+- procstat_[prefix_]rlimit_nice_priority_hard value=0
+- procstat_[prefix_]rlimit_nice_priority_soft value=0
+- procstat_[prefix_]rlimit_num_fds_hard value=16384
+- procstat_[prefix_]rlimit_num_fds_soft value=16384
+- procstat_[prefix_]rlimit_realtime_priority_hard value=0
+- procstat_[prefix_]rlimit_realtime_priority_soft value=0
+- procstat_[prefix_]rlimit_signals_pending_hard value=1758
+- procstat_[prefix_]rlimit_signals_pending_soft value=1758
+
+*NOTE: Due to a limitation in an underlying library Telegraf uses, any resource limit > 2147483647 will be misreported as 2147483647.*
```

Comment:
Oh, and I just remembered to add this little note. I think this is a major issue. But the `gopsutil` owners won't fix the issue because it might break consumers. (which I personally think is a lousy excuse as that's what vendoring is meant to solve)